recipe/meta.yaml

```diff
@@ -1,43 +1,53 @@
-{% set version = "1.2.2" %}
+{% set version = "1.2.3" %}
```

Comment:
Could you update to the latest **v1.2.4**? See diff https://github.com/IntelPython/mkl_random/compare/v1.2.3...v1.2.4